test/org/sagebionetworks/bridge/play/controllers/ParticipantControllerTest.java

```diff
@@ -196,18 +199,43 @@ public void before() throws Exception {
     public void getParticipants() throws Exception {
         DateTime start = DateTime.now();
         DateTime end = DateTime.now();
-        Result result = controller.getParticipants("10", "20", "foo", start.toString(), end.toString());
+        Result result = controller.getParticipants("10", "20", "foo", start.toString(), end.toString(), null, null);
         PagedResourceList<AccountSummary> page = resultToPage(result);
         
         // verify the result contains items
         assertEquals(3, page.getItems().size());
-        assertEquals(30, page.getTotal());
+        assertEquals((Integer)30, page.getTotal());
         assertEquals(SUMMARY, page.getItems().get(0));
         
         //verify paging/filtering
-        assertEquals(new Integer(10), page.getOffsetBy());
-        assertEquals(20, page.getPageSize());
-        assertEquals("foo", page.getFilters().get("emailFilter"));
+        assertEquals((Integer)10, page.getRequestParams().get("offsetBy"));
+        assertEquals(20, page.getRequestParams().get("pageSize"));
+        assertEquals("foo", page.getRequestParams().get("emailFilter"));
+        
+        // DateTime instances don't seem to be equal unless you use the library's equality methods, which
+        // verification does not do. So capture and compare that way.
+        verify(mockParticipantService).getPagedAccountSummaries(eq(study), eq(10), eq(20), eq("foo"),
+                startTimeCaptor.capture(), endTimeCaptor.capture());
+        assertEquals(start.toString(), startTimeCaptor.getValue().toString());
+        assertEquals(end.toString(), endTimeCaptor.getValue().toString());
+    }
+    
+    @Test
+    public void getParticipantsWithStartTimeEndTime() throws Exception {
+        DateTime start = DateTime.now();
+        DateTime end = DateTime.now();
```

Comment:
Chances are, start and end will be the same. Would it be better to define end as start.plusMinutes(1) or something like that?  (Similarly for getParticipantsForWorker().)